pkg/rewards/claim.go

```diff
@@ -83,6 +85,10 @@ func Claim(cCtx *cli.Context, p utils.Prompter) error {
 	ctx := cCtx.Context
 	logger := common.GetLogger(cCtx)
```

Comment:
why don't we add the SilentFlag logic in the `GetLogger` function. that way eventually we extend this to other commands?